cmd/podman/play/kube.go

```diff
@@ -150,6 +155,11 @@ func kube(cmd *cobra.Command, args []string) error {
 	if kubeOptions.Down {
 		return teardown(yamlfile)
 	}
+	if kubeOptions.Replace {
+		if err := teardown(yamlfile); err != nil && !errorhandling.Contains(err, define.ErrNoSuchPod) {
```

Comment:
the delete pod API of podman remote service returns untyped errors, so use string compare here.